README.md

```diff
@@ -113,6 +113,7 @@ fly \
   -v state_git_repo=<PROVIDE>
   -v github-private-key=<PROVIDE> \
   -v bosh_lite_name=<PROVIDE> \
+  -v domain_name=<PROVIDE>
```

Comment:
This is basically the same as `sl_vm_domain`. We could either make that clear by using the same name, or we should at least document that in the list below.